k8s-deploy/pkg/cli/cluster_cli.go

```diff
@@ -162,14 +162,13 @@ func ClusterInstallCommand() *cli.Command {
 			if !ok || chartName == "" {
 				return errors.New("chartName not found, check your cluster.yaml file")
 			}
-			if template, err := GetValueTemplateExample(chartName.(string), chartVersion.(string)); err != nil {
-				if _, ok := err.(VersionNotValidError); !ok {
-					// if the error is not VersionNotValidError, warn user.
-					color.Yellow("Config Warning: %s\n", err.Error())
-				}
-			} else {
-				skippedKeys := getSkippedKeys(m)
-				errs := ValidateYaml(template, string(clusterConfig), skippedKeys)
+
+			var errs []error
+			template, err := GetValueTemplateExample(chartName.(string), chartVersion.(string))
+			errs = append(errs, err)
+			skippedKeys := append(getSkippedKeys(m), "route_table")
+			errs = append(errs, ValidateYaml(template, string(clusterConfig), skippedKeys, nil)...)
+			if !ContainsSkipError(errs) {
 				for _, err := range errs {
 					if err != nil {
 						color.Yellow("Config Warning: %s\n", err.Error())
```

Comment:
https://github.com/FederatedAI/KubeFATE/blob/master/k8s-deploy/kubefate.yaml#L64-L65

Can this configuration control the log with or without color?